pkg/unpriv/unpriv.go

```diff
@@ -315,6 +320,7 @@ func Remove(path string) error {
 // currently have enough access bits.
 func RemoveAll(path string) error {
 	return errors.Wrap(Wrap(path, func(path string) error {
+		// XXX(VR): os.RemoveAll(path) works, but I am not sure if that's right
 		// If remove works, we're done.
 		err := os.Remove(path)
```

Comment:
So, if we use os.RemoveAll(path) here, the CI succeeds. Would this be an alternative here, @cyphar?